src/components/describe/describeWebview.ts

```diff
@@ -0,0 +1,91 @@
+import * as vscode from 'vscode';
+
+export class DescribePanel {
+	public static readonly viewType = 'vscodeKubernetesDescribe';
+	public static currentPanels = [];
+
+	private readonly panel: vscode.WebviewPanel;
+    private readonly extensionPath: string;
+    private content: string;
+	private resource: string;
+
+	private disposables: vscode.Disposable[] = [];
+
+	public static createOrShow(extensionPath: string, content: string, resource: string) {
```

Comment:
Seems like callers only ever pass `null` for extensionPath.  Is it used?  Can we remove it for now?

Reply:
removed for now. `extensionPath` is needed if we ever want to load scripts/stylesheets for the webview, but we can cross that bridge when we come to it.